Automattic/wp-calypso
# Launchpad: Remove unscoped css style from Celebrate Launch modal

This is a follow up to this PR https://github.com/Automattic/wp-calypso/pull/75275#discussion_r1160660752

### Time Estimate
Review: Short
Test: Short

### Summary
This PR removes unscoped styles from `celebrate-launch.scss` to the tooltip component's own stylesheet to avoid unexpected style bleed.

### Testing
1. Checkout this branch
2. Create a new Launchpad site and proceed to launch it
3. Once you get redirected to My Home you will see the celebrate launch modal appear
4. Click the Copy site button and confirm you still see the tooltip **Copied to clipboard!**

![image](https://user-images.githubusercontent.com/20927667/231568802-578d2652-3e98-4cf0-a7cb-48621140b3cb.png)
